src/Security/Authorization/test/AuthorizationMiddlewareTests.cs

```diff
@@ -374,6 +376,55 @@ public async Task CanApplyPolicyDirectlyToEndpoint()
         Assert.True(calledPolicy);
     }
 
+    public class ReqAttribute : Attribute, IRequirementData
```

Comment:
Add an example that does:

```C#
public class ReqAttribute2 : AuthorizeAttribute, IRequirementData
```

Reply:
Updated and added the CanApplyAuthorizeRequirementAttributeDirectlyToEndpoint test below for this